docs/how_to_guides/envvars.md

```diff
@@ -1,4 +1,4 @@
-# Set API Keys and other Environment Variables
+# Set API Keys and other EnvVars
```

Comment:
Why abbreviate here?

Reply:
@CalebCourier helps the page title fit in a single line in the left sidebar menu. Not abbreviated below for clarity.